packages/hopli/Dockerfile

```diff
@@ -4,35 +4,37 @@ ARG DEBIAN_IMAGE=${DEBIAN_IMAGE:-debian:bullseye-slim@sha256:64e251208f3bddf166a
 
 FROM ${HOPR_TOOLCHAIN_IMAGE} as builder
 
-WORKDIR /app/hopli
+WORKDIR /app/hoprnet
 
 # musl toolchain for static binaries
 RUN apt update && apt install -y pkg-config libssl-dev musl-tools
 ENV SYSROOT=/dummy
 ENV OPENSSL_STATIC=1
 ENV OPENSSL_INCLUDE_DIR=/usr/include/openssl
 
-COPY ./packages/hopli/Cargo.toml ./Cargo.toml
-COPY ./packages/hopli/src ./src
+COPY ./Cargo.toml ./Cargo.toml
+COPY packages packages/
+
+WORKDIR /app/hoprnet/packages/hopli
 
 RUN rustup target install $(uname -m)-unknown-linux-musl
-RUN OPENSSL_LIB_DIR=/usr/lib/$(uname -m)-linux-gnu RUSTFLAGS="-C target-feature=+crt-static" cargo build --release --target $(uname -m)-unknown-linux-musl
+RUN OPENSSL_LIB_DIR=/usr/lib/$(uname -m)-linux-gnu RUSTFLAGS="-C target-feature=+crt-static" cargo build --release --target $(uname -m)-unknown-linux-musl --target-dir /app/hoprnet/packages/hopli/target
```

Comment:
This should not be necessary, since the previous workdir sets this directory as root anyways.

Reply:
Apparently it shouldn't but if you check the previous build failures https://github.com/hoprnet/hoprnet/actions/runs/4752313469/jobs/8442524703 and this flag solved the issue.

Reply:
It looks like the next change in COPY step fixed it.